tests/test_forms.py

```diff
@@ -1580,8 +1584,8 @@ async def test_form_validation_action():
 
     assert not warnings
     assert events == [
-        SlotSet("slot2", None),
         SlotSet("slot1", "validated_value"),
+        SlotSet("slot2", "slot1 == validated_value"),
```

Comment:
I agree that the order of slots needs to be changed in the list after the change you proposed - however the test should work with `SlotSet("slot2", None)`